translations/harbour-berlin-vegan-nl.ts

```diff
@@ -362,5 +354,11 @@
         <source>closes soon</source>
         <translation>sluit snel</translation>
     </message>
+    <message id="id-venue-please-tell-waiter">
+        <location filename="../qml/pages/VenueDescription.qml" line="262"/>
+        <source>Please tell the waiter/owner that you found their venue via the Berlin-Vegan app. The significance of being listed in a database like this is often underestimated. Thanks for the help!</source>
+        <oldsource>Please tell the waiter/owner that you found their venue via the Berlin-Vegan app.</oldsource>
+        <translation>Vertel de ober / eigenaar dat je zijn locatie gevonden hebt via de app Berlijn-Vegan. Het belang om in zo&apos;n database te worden vermeld, wordt vaak onderschat. Bedankt voor de hulp!</translation>
```

Comment:
@Vistaus Hello Heimen, can you check that please?

Reply:
Hi @micuintus,

Almost perfect, but I corrected a few small things :)

"Vertel de ober/eigenaar dat je zijn locatie gevonden hebt via de app Berlijn-Vegan. Het belang om in zo'n databank te worden vermeld wordt vaak onderschat. Bedankt voor de hulp!"

Reply:
Cool Thanks @Vistaus, incorporated it!

Reply:
No problem :)